packages/context/src/context.ts

```diff
@@ -66,6 +66,153 @@ export class Context {
     return binding;
   }
 
+  /**
+   * Create a corresponding binding for configuration of the target bound by
+   * the given key in the context.
+   *
+   * For example, `ctx.configure('controllers.MyController', {x: 1})` will
+   * create binding `controllers.MyController:$config` with value `{x: 1}`.
+   *
+   * @param key The key for the binding that accepts the config
+   * @param config Options object
+   */
+  configure(key: string, config?: BoundValue): Binding {
```

Comment:
This creates inconsistency with `ctx.bind` API, because `ctx.bind` does not accept the binding value as an optional argument.

Reply:
Fixed - remove the optional `config` argument